src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -783,6 +792,22 @@ bool EnzymeTreeItem::operator<(const QTreeWidgetItem& other) const {
     return text(col) < ei.text(col);
 }
 
+QString EnzymeTreeItem::getEnzymeInfo() const {
+    QString result;
+    auto accession = text(Column::Accession);
+    accession = accession.right(accession.size() - 2);
```

Comment:
Use accession in the link. Otherwise it is an unused variable.
Check that accession is RB (rebase) before opening the link.